### PR TITLE
Set 5 second timeout for cluster server connection that requires VPN and code clean-up

### PR DESCRIPTION
**Pull request by yemi-kelani**

### CODE REVIEW

File: src/kubernetes/kubernetes.ts [hunk 1]
```diff
@@ -616,6 +616,31 @@ export class KubernetesObj extends KubernetesContext {
...
         return false;
       });
+
+    // Cancel request after 5 seconds without a response from the readNamespace request.
+    // This usually implies a connectivity issue with OpenShift, which could take a minute or more
+    // before receiving the timeout response.
+    let timeout: NodeJS.Timeout | undefined = undefined;
+    const timeoutPromise: Promise<boolean> = new Promise(resolve => {
+      timeout = setTimeout(() => {
+        vscode.window.showWarningMessage('Connection timed out on "validateNamespaceExists"... Please check your internet/VPN connection.');
```
Comment (freemanlatrell) [inline src/kubernetes/kubernetes.ts:626]: Since this message is displayed to the end user, can you remove the function name from this message? Also do you think it's useful to the user to know where in the validation that it's failing versus just knowing that there's a connectivity issue? Lastly, you can also remove the term `internet/VPN` and reword it to something like this:

"Connection timed out...Please validate the connectivity to OpenShift"

File: src/utilities/session.ts [hunk 2]
```diff
@@ -128,32 +128,35 @@ export class Session {
...
 
     // Cancel request after 5 seconds without a response from the getNamespaceList request.
     // This usually implies a connectivity issue with OpenShift, which could take a minute or more
     // before receiving the timeout response.
-    const timeout: Promise<boolean> = new Promise(resolve => {
-      setTimeout(() => {
+    let timeout: NodeJS.Timeout | undefined = undefined;
+    const timeoutPromise: Promise<boolean> = new Promise(resolve => {
+      timeout = setTimeout(() => {
+        vscode.window.showWarningMessage('Connection timed out on "getNamespaceList"... Please check your internet/VPN connection.');
```
Comment (freemanlatrell) [inline src/utilities/session.ts:142]: Since this message is displayed to the end user, can you remove the function name from this message? Also do you think it's useful to the user to know where in the validation that it's failing versus just knowing that there's a connectivity issue? Lastly, you can also remove the term internet/VPN and reword it to something like this:

"Connection timed out...Please validate the connectivity to OpenShift"

**Review by freemanlatrell — CHANGES_REQUESTED** [review]

File: src/kubernetes/kubernetes.ts (hunk 1 above)
Comment (yemi-kelani) [reply src/kubernetes/kubernetes.ts:626]: Good call, probably not useful for the user to know. More-so for debugging.
